docs/scripts/buildApiUtils.ts

```diff
@@ -48,7 +48,7 @@ const packages = [
 
 packages.forEach((pkg) => {
   pkg.paths.forEach((pkgPath) => {
-    const packageName = pkgPath.match(/packages\/([^/]+)\/src/)?.[1];
+    const packageName = pkgPath.match(/packages(\\|\/)([^/]+)(\\|\/)src/)?.[1];
```

Comment:
This change introduced new capturing groups, so `match(...)[1]` doesn't refer to the same thing anymore.

Additionally, let's make it consistent and exclude both delimiters in the package name.

```suggestion
    const packageName = pkgPath.match(/packages(?:\\|\/)([^/\\]+)(?:\\|\/)src/)?.[1];
```

Reply:
Thanks. Now the first index is the package name like `mui-lab`, `mui-material` etc and the package also excludes both delimiters.